content/en/docs/instrumentation/erlang/_index.md

```diff
@@ -2,7 +2,9 @@
 title: Erlang/Elixir
 weight: 14
 description: >
-  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Elixir_SDK.svg"></img>
+  <img width="35"
+  src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Elixir_SDK.svg"
```

Comment:
can you use the Erlang logo here? (should just be s/Elixir/Erlang) cc: @tsloughter